src/style.css

```
@@ -0,0 +1,8 @@
+@import url("./cardStyles/mainAndHtml.css");
+@import url("./cardStyles/card.css");
+@import url("./cardStyles/cardPicture.css");
+@import url("./cardStyles/cardMainText.css");
+@import url("./cardStyles/cardProductCode.css");
+@import url("./cardStarReviews.css");
+@import url("./cardStyles/cardPrice.css");
+@import url("./cardStyles/cardButton.css");

```

```suggestion
@import url("./cardStyles/base.css");
@import url("./cardStyles/card.css");
@import url("./cardStyles/stars.css");
```
the structure should look like this